client/src/pages/demo/components/index.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/no-unescaped-entities */
+import React, { Fragment } from 'react';
+import Iframe from '@/src/shared/components/Iframe';
+
+const Index: React.FunctionComponent = () => {
```

Comment:
- change to specific component name
```suggestion
const DemoComponent: React.FunctionComponent = () => {
```